README.md

```diff
@@ -7,19 +7,21 @@ Yet another F-Droid client.
 
 ## Description
 
-Unofficial F-Droid client that resembles the classic F-Droid client.
+Unofficial F-Droid client resembling the classic F-Droid client.
 
-The client supports all basic F-Droid features, such as repository management and package installation. It doesn't
-support privileged extension, root installation, or local repositories.
+Jump over the lazy dog, manage repositories and install software quickly.
+No privileged extension, root installation, or sharing local repositories nearby yet.
```

Comment:
```suggestion
No privileged extension, root installation, or sharing local repositories nearby.
```
Maybe with https://f-droid.org/en/packages/com.genonbeta.TrebleShot/ sometime in the future.